ngraph/core/include/ngraph/op/lstm_sequence.hpp

```diff
@@ -62,93 +61,34 @@ namespace ngraph
                                       const Output<Node>& W,
                                       const Output<Node>& R,
                                       const Output<Node>& B,
-                                      const Output<Node>& P,
                                       const std::int64_t hidden_size,
                                       const direction lstm_direction,
-                                      LSTMWeightsFormat weights_format = LSTMWeightsFormat::IFCO,
```

Comment:
Current ONNX importer LSTM implementation just passes the `IOFC` format (ONNX standard) by the `ngraph::op::LSTMWeightsFormat` to the LSTMSequence while the [OpenVINO LSTMSequence](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/LSTMSequence_1.md) supports only `FICO`. 
We need a way to align the weights format.

Reply:
implemented ngraph/core/include/ngraph/op/util/rnn_cell_base.hpp/cpp, used in lstm_cell_zero_bias_peepholes